dask/array/tests/test_array_core.py

```diff
@@ -3032,3 +3032,14 @@ def test_delayed_array_key_hygeine():
     d = delayed(identity)(a)
     b = da.from_delayed(d, shape=a.shape, dtype=a.dtype)
     assert_eq(a, b)
+
+
+def test_empty_chunks_in_array_len():
+    da = pytest.importorskip('dask.array')
```

Comment:
This is unnecessary, this is checked at the top of the file.

Reply:
Oops. Fixed!